samples/hello-js/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ import org.jetbrains.kotlin.gradle.tasks.Kotlin2JsCompile
 
 buildscript {
     dependencies {
-        classpath(kotlin("gradle-plugin"))
+        classpath(kotlin("gradle-plugin", "1.1.50"))
```

Comment:
:x: We should also introduce test cases to make sure we don't forget to update the version whenever we bump the embedded Kotlin version.

Reply:
Done